test/test.js

```diff
@@ -157,14 +157,22 @@ describe('cypress-terminal-report', () => {
     });
   }).timeout(60000);
 
-  it('Should always print logs when configuration enabled.', async () => {
-    await runTest(commandBase(['printLogsAlways=1'], ['alwaysPrintLogs.spec.js']), (error, stdout, stderr) => {
+  it('Should always print logs to console when configured so.', async () => {
+    await runTest(commandBase(['printLogsToConsoleAlways=1'], ['printLogs.spec.js']), (error, stdout, stderr) => {
       // cy.command logs.
       expect(stdout).to.contain(`cy:command ${ICONS.success}  visit\t/\n`);
       expect(stdout).to.contain(`cy:command ${ICONS.success}  contains\tcypress\n`);
     });
   }).timeout(60000);
 
+  it('Should never print logs to console when configured so.', async () => {
+    await runTest(commandBase(['printLogsToConsoleNever=1'], ['printLogs.spec.js']), (error, stdout, stderr) => {
```

Comment:
Here we should run two specs: one that is failing and one that is passing to properly test that it never outputs. Same for when writing to file.